src/AjaxDynamicDataTable.jsx

```diff
@@ -86,13 +92,16 @@ class AjaxDynamicDataTable extends Component {
     loadPage(page) {
         const {perPage, orderByField, orderByDirection} = this.state;
         const {onLoad, params, axios} = this.props;
-
         this.setState(
             { loading: true },
             () => {
+                const CancelToken = axios.CancelToken;
+                this.cancelRequest=function(){};
+                const thisComponent=this;
                 axios.get(this.props.apiUrl, {
 
-                    params: { ...params, page, perPage, orderByField, orderByDirection }
+                    params: { ...params, page, perPage, orderByField, orderByDirection },
+                    cancelToken: new CancelToken(function executor(c){thisComponent.cancelRequest=c;})
```

Comment:
If you use an anonymous function here then you wouldn't have to define `thisComponent`.

```suggestion
                    cancelToken: new CancelToken(callback => this.cancelRequest = callback)
```